modules/tensor_mechanics/test/tests/shell/static/tapered.i

```diff
@@ -0,0 +1,348 @@
+    execute_on = TIMESTEP_END
+  []
+  [stress_10]
+    type = RankTwoAux
+    variable = stress_10
+    rank_two_tensor = global_stress_t_points_0
+    index_i = 1
+    index_j = 0
+    execute_on = TIMESTEP_END
+  []
+  [strain_10]
+    type = RankTwoAux
+    variable = strain_10
+    rank_two_tensor = total_global_strain_t_points_0
+    index_i = 1
+    index_j = 0
+    execute_on = TIMESTEP_END
+  []
+  [stress_02]
+    type = RankTwoAux
+    variable = stress_02
+    rank_two_tensor = global_stress_t_points_0
+    index_i = 0
+    index_j = 2
+    execute_on = TIMESTEP_END
+  []
+  [strain_02]
+    type = RankTwoAux
+    variable = strain_02
+    rank_two_tensor = total_global_strain_t_points_0
+    index_i = 0
+    index_j = 2
+    execute_on = TIMESTEP_END
+  []
+  [stress_20]
+    type = RankTwoAux
+    variable = stress_20
+    rank_two_tensor = global_stress_t_points_0
+    index_i = 2
+    index_j = 0
+    execute_on = TIMESTEP_END
+  []
+  [strain_20]
+    type = RankTwoAux
+    variable = strain_20
+    rank_two_tensor = total_global_strain_t_points_0
+    index_i = 2
+    index_j = 0
+    execute_on = TIMESTEP_END
+  []
+  [stress_12]
+    type = RankTwoAux
+    variable = stress_12
+    rank_two_tensor = global_stress_t_points_0
+    index_i = 1
+    index_j = 2
+    execute_on = TIMESTEP_END
+  []
+  [strain_12]
+    type = RankTwoAux
+    variable = strain_12
+    rank_two_tensor = total_global_strain_t_points_0
+    index_i = 1
+    index_j = 2
+    execute_on = TIMESTEP_END
+  []
+  [stress_21]
+    type = RankTwoAux
+    variable = stress_21
+    rank_two_tensor = global_stress_t_points_0
+    index_i = 2
+    index_j = 1
+    execute_on = TIMESTEP_END
+  []
+  [strain_21]
+    type = RankTwoAux
+    variable = strain_21
+    rank_two_tensor = total_global_strain_t_points_0
+    index_i = 2
+    index_j = 1
+    execute_on = TIMESTEP_END
+  []
+[]
+
+[BCs]
+  [BC_0]
+    type = ADDirichletBC
+    variable = disp_x
+    value = 0.0
+    boundary = '2' #left
+  []
+  [BC_1]
+    type = ADDirichletBC
+    variable = disp_y
+    value = 0.0
+    boundary = 10 #left_side_mid
+  []
+  [BC_2]
+    type = ADPressure
+    variable = disp_x
+    displacements = 'disp_x disp_y disp_z'
+    factor = -50
+    boundary = '3'
+    use_displaced_mesh = false
+  []
+[]
+
+[Materials]
+  [stress]
+    type = ADComputeShellStress
+    block = 1
+    through_thickness_order = SECOND
+  []
+  [elasticity]
+    type = ADComputeIsotropicElasticityTensorShell
+    youngs_modulus = 210000
+    poissons_ratio = 0.3
+    block = 1
+    through_thickness_order = SECOND
+  []
+  [strain]
+    type = ADComputeIncrementalShellStrain
+    block = 1
+    displacements = 'disp_x disp_y disp_z'
+    rotations = 'rot_x rot_y'
+    thickness = 0.1
+    through_thickness_order = SECOND
+  []
+[]
+
+[Executioner]
+  type = Steady
+  solve_type = NEWTON
+  nl_abs_tol = 1e-8
+  l_abs_tol = 1e-8
+  nl_max_its = 100
+  l_max_its = 100
+[]
+
+[Outputs]
+  [outfile]
```

Comment:
`exodus=true` rather than this sub-block.

Reply:
Removed subblock.